libs/connector/src/secretarium.connector.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { assert } from 'console';
+import { SCP } from './secretarium.connector';
+import { Key } from './secretarium.key';
+// import { ConnectionStateMessage } from './secretarium.constant';
+
+describe('Connection test', () => {
+    it('Connects to Klave', async () => {
+        const scp = new SCP();
+        console.log('\n>> Opening SCP...');
+        await scp.connect();
+
+        const tx = scp.newTx('wasm-manager', 'version', 'requestId1', { });
+        tx.onError((err) => { console.error('Query error:', err); });
+        tx.onResult((res) => { console.log('Query result:', res); });
```

Comment:
In tests these shouldn't print to the terminal. `console.log` should be a mock

Reply:
There is also no asynchronous trailing for the test to close ... so this will trigger memory-leak in CI